test-suite/js/src/test/scala/org/scalajs/testsuite/javalib/io/DataInputStreamTest.scala

```diff
@@ -0,0 +1,42 @@
+package org.scalajs.testsuite.javalib.io
```

Comment:
This file should be called `DataInputStreamJSTest` (with `JS`), to be consistent with other tests that have a JS-specific part.